src/bin/resource_tracing.rs

```diff
@@ -337,7 +345,10 @@ struct RoomInfo
 {
     room_id: u32,
     name: String,
+    name_id: u32,
+    mapa_id: u32,
```

Comment:
These should probably be `ResId<STRG>` and `ResId<MAPA>`. `room_id` too should probably have been udpated to ``ResId<MREA>`.